vrs/marten-construction-script.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Daniel Marten 01-04-2022
+# Constructing a ~100k Variant Test Database for GA4GH-VRS Annotation Script
+# NOTE: adapted from a Jupyter Notebook, so excuse any formatting quirks! 
+# NOTE: please make sure to change the paths for checkpointing!!!
+# Edits on: 01-06-2022
+
+import hail as hl
+import random
+from random import sample
+from gnomad.resources.grch38.gnomad import public_release
+
+# Setting randomness
+random.seed(505)
+hl.init(global_seed=5)
+
+# Import the gnomAD v3.1.2 full variant table (approx 759,000,000 variants)
+
+ht_whole = public_release("genomes").ht()
+ht_whole = ht_whole.naive_coalesce(10_000)
+
+# PREVIOUSLY CALCULATED FREQUENCY STATISTICS: 
+# Frequency of the following in the general v3 database, based off of frequencies in a randomly sampled
+# database of 1mil variants
+# for: indels, long references (>5bp), long variants (>5bp), x-chromosome variants, y-chromosome variants
+# if you would like, uncomment and run the below code: 
+
+# # Generating Frequencies: 
+# ht_1mil = ht_whole.sample(0.0013175) # into 1million samples == 1_000_000/759_000_000 (whole count)
+# ht_1mil = ht_1mil.checkpoint('gs://gnomad-tmp-4day/marten-temp-01-06/sample_1mil.ht',overwrite=True)
+# count_1mil = ht_1mil.count()
+# indel_freq = ht_1mil.filter(hl.is_indel(ht_1mil.alleles[0], ht_1mil.alleles[1])).count() / count_1mil # = 0.14531311994611185
+# ref_freq = ht_1mil.filter(hl.len(ht_1mil.alleles[0])>5).count() / count_1mil # = 0.02129941204747994
+# var_freq = ht_1mil.filter(hl.len(ht_1mil.alleles[1])>5).count() / count_1mil # = 0.03805950683146261
+# x_freq = ht_1mil.filter(ht_1mil.locus.contig == "chrX").count() / count_1mil # = 0.04005632671091385
+# y_freq = ht_1mil.filter(ht_1mil.locus.contig == "chrY").count() / count_1mil # = 0.0015570797658583776
+# print(f'indel freq as: {indel_freq} \n\tlong ref and var freq as: {ref_freq} , {var_freq} \n\tx and y as: {x_freq} , {y_freq}')
+
+indel_freq = 0.1453
+ref_freq = 0.02130 
+var_freq = 0.03806 
+x_freq = 0.04006 
+y_freq = 0.001557 
+whole_count = 759_000_000
```

Comment:
```suggestion
whole_count = ht_whole.count()
```
it's good practice to avoid hardcoding information where possible (keeps the code more flexible/general)